client/asset/btc/btc.go

```diff
@@ -816,6 +827,10 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, net dex.Network) (ass
 			return nil, err
 		}
 		return &ExchangeWalletAccelerator{rpcWallet}, nil
+	case walletTypeElectrum:
+		cloneCFG.SingularWallet = true     // not used, but no path on RPC endpoint
```

Comment:
Sorry, this comment was confusing.  The field name is still pretty good, my comment just makes no sense.  I'll just remove this line since the actual code is ineffectual because the request code does not use this variable, instead just doing `"http://"+rpcCfg.RPCBind` in `func ElectrumWallet`.